### ISSUE TITLE
Dúvidas

### ISSUE THREAD

**Issue by danilosilvadev** [training-center/meetup-clone#3]
Tive a ideia de criar essa issue que pode ser um lugar pra perguntarmos sobre o projeto, partes dele e sobre implementação de libs, react e etc. Quem tiver dúvidas escreva abaixo e vamo respondendo. Não sei se eu podia criar issue mas tá ae, qualquer coisa me avisem hehe.

**Comment by woliveiras** [comment]
> Não sei se eu podia criar issue mas tá ae

HHIAHIAHI

Sempre pode mano. A comunidade é nossa! 

Pode fazer qualquer coisa que não seja infração ao código de conduta. :)

Mas não seria melhor criar uma issue por dúvida quando as pessoas perguntarem?

Assim fica fácil de localizar pela busca.

Uma issue geral pode ficar tão inflada ou tão específica que atrapalha uma futura consulta.

**Comment by fdaciuk** [comment]
Concordo com o @woliveiras! Dúvidas são sempre válidas, e especialmente no caso desse projeto, é um caso de estudo mesmo, mas tentando de alguma forma ser útil em um futuro próximo =)

Então, acho que dúvidas específicas, por issue, fazem mais sentido. Até porque aqui não tem thread pra organizar xD

**Comment by rafaelbezerra195** [comment]
Acho q é isso mesmo, dúvidas por Issue, acho q uma Label já ajuda a identificar.
